### PR TITLE
Fix: check kube-proxy's 10256 port healthz

### PR DESCRIPTION
**Pull request by xieyanker**
Fix https://github.com/alauda/kube-ovn/issues/654

### CODE REVIEW

File: dist/images/kubectl-ko [hunk 1]
```diff
@@ -263,6 +263,14 @@ checkDeployment(){
   fi
 }
 
+checkKubeProxy(){
+  healthResult=`kubectl get node -o wide | grep -v "INTERNAL-IP" | awk '{printf "curl -sL -w %{http_code} http://%s:10256/healthz -o /dev/null | grep -v 200 \n", $6}' | bash` || true
```
Comment (oilbeater) [inline dist/images/kubectl-ko:267]: ipv6 e2e test failed. Should use `curl -g -6` for ipv6 cluster

**Review by oilbeater — CHANGES_REQUESTED** [review]

File: dist/images/kubectl-ko (hunk 1 above)
Comment (xieyanker) [reply dist/images/kubectl-ko:267]: @oilbeater Thanks for your reply! However, I don’t have an ipv6 cluster. What I want to know is whether `curl -g -6` is also valid for ipv4 cluster. Should I need different commands for these two clusters?

Comment (oilbeater) [reply dist/images/kubectl-ko:267]: After some tests, these options also work for ipv4 cluster. We can go on

File: dist/images/install-pre-1.16.sh [hunk 2]
```diff
@@ -2083,6 +2083,14 @@ checkDaemonSet(){
   fi
 }
 
+checkKubeProxy(){
+  healthResult=`kubectl get node -o wide | grep -v "INTERNAL-IP" | awk '{printf "curl -g -6 -sL -w %{http_code} http://%s:10256/healthz -o /dev/null | grep -v 200 \n", $6}' | bash` || true
```
Comment (oilbeater) [inline dist/images/install-pre-1.16.sh:2087]: `http://[%s]:10256/healthz` square brackets is needded

**Review by oilbeater — CHANGES_REQUESTED** [review]

**Review by oilbeater — APPROVED** [review]

**Comment by oilbeater** [comment]
Thanks! @xieyanker

**Pull request merged**